src/core/ext/transport/inproc/inproc_transport.cc

```diff
@@ -193,10 +288,10 @@ static void log_metadata(const grpc_metadata_batch* md_batch, bool is_client,
   }
 }
 
-static grpc_error* fill_in_metadata(inproc_stream* s,
-                                    const grpc_metadata_batch* metadata,
-                                    uint32_t flags, grpc_metadata_batch* out_md,
-                                    uint32_t* outflags, bool* markfilled) {
+grpc_error* fill_in_metadata(inproc_stream* s,
```

Comment:
Was there a reason for this to lose its static?

Reply:
The sole reason being my clumsiness. Sorry about that. Thank for catching this. I moved everything to anonymous namespace now.